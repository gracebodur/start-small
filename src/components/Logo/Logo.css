.Logo {
    display: inline-block;
    margin: 10px;
    margin-top: 0px;
    padding: 5px;
    border-radius: 2px;
}