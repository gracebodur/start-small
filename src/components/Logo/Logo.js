import React from 'react'
import {Link} from 'react-router-dom'
import Tilt from 'react-tilt'
import logo from '../../img/logo_file.png'
import './Logo.css'


function Logo() {
    return(
        <div className='Logo'>
            <Tilt className="Tilt" options={{ max : 25 }} style={{ height: 100, width: 100 }}>
                <Link to='/' className="Tilt-inner">
                    <img style={{paddingTop: '5px'}} alt='logo' src={logo} />
                </Link>
            </Tilt>
        </div>
    )
}

export default Logo