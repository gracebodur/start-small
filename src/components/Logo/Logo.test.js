import React from "react";
import toJson from "enzyme-to-json";
import Logo from "./Logo";

describe(`Logo component`, () => {
  const props = {
    id: "a",
    name: "test-class-name"
  };

  it("renders a Logo by default", () => {
    const wrapper = <Logo />;
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it("renders the Logo given props", () => {
    const wrapper = <Logo {...props} />;
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
