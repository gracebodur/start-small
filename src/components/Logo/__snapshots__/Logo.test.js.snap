// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component renders a Logo by default 1`] = `undefined`;

exports[`Logo component renders the Logo given props 1`] = `undefined`;
